packages/@apphosting/adapter-nextjs/src/utils.ts

```diff
@@ -96,23 +98,28 @@ export async function generateOutputDirectory(
   await move(standaloneDirectory, outputBundleOptions.outputDirectory, { overwrite: true });
 
   const staticDirectory = join(nextBuildDirectory, "static");
-  const publicDirectory = join(appDir, "public");
   await Promise.all([
     move(staticDirectory, outputBundleOptions.outputStaticDirectory, { overwrite: true }),
-    movePublicDirectory(publicDirectory, outputBundleOptions.outputPublicDirectory),
+    moveResources(appDir, outputBundleOptions.outputBundleAppDir),
     generateBundleYaml(outputBundleOptions, nextBuildDirectory, rootDir),
   ]);
   return;
 }
 
-// move public directory to apphosting output public directory
-async function movePublicDirectory(
-  publicDirectory: string,
-  appHostingPublicDirectory: string,
-): Promise<void> {
-  const publicDirectoryExists = await exists(publicDirectory);
-  if (!publicDirectoryExists) return;
-  await move(publicDirectory, appHostingPublicDirectory, { overwrite: true });
+// Move all files and directories to apphosting output directory.
+// Files are skipped if there is already a file with the same name in the output directory
+async function moveResources(appDir: string, outputBundleAppDir: string): Promise<void> {
+  const appDirExists = await exists(appDir);
+  if (!appDirExists) return;
+  const dirsToMove = await readdir(appDir);
```

Comment:
Are these results all guaranteed to be directories?

Reply:
good catch, they can be files or dirs. I renamed the variable to account for this